src/libstd/fs.rs

```diff
@@ -1090,13 +1090,14 @@ impl Metadata {
 
     /// Returns the creation time listed in this metadata.
     ///
-    /// The returned value corresponds to the `birthtime` field of `stat` on
+    /// The returned value corresponds to the `btime` field of `statx` on
+    /// Linux not prior to 4.11, the `birthtime` field of `stat` on other
```

Comment:
"not prior" is hard to parse; what about "starting from" or so?

You are basically saying `!(linux_version < (4, 11))` instead of the much more comprehensible `linux_version >= (4, 11)`.

Reply:
Fixed